project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala

```diff
@@ -53,12 +53,21 @@ object Docker {
       label("CLIO_VERSION", version.value)
       add(artifact, artifactTargetPath)
       expose(8080, 31757)
+
       /*
        * We need to use `entryPointShell` here to allow the environment variable to be substituted.
        * By default, using the shell form of ENTRYPOINT prevents signals from reaching the docker container.
        * Prepending `exec` fixes the problem.
        */
-      entryPointShell("exec", "java", "$JAVA_OPTS", "-jar", artifactTargetPath)
+      entryPointShell(
+        "umask",
+        "$UMASK;",
```

Comment:
Not sure how this works, is this building a command line out of the arguments? I would use `&&` instead of `;` to sequence commands in sh.

Since there are only two states here it might be simpler to use a boolean and have this code hardcode the two different umasks we want to support.

Reply:
We don't really care if umask fails.. if it does it goes back to default behavior.
Likewise if it isn't set it will just print out and use the default umask value. 
The two states are just set it to something or use the default.

Reply:
OK I get it. Still, it does seem odd to include two different command lines when calling `entryPointShell()`. But if it works, it works. Have you started an integration build with this branch yet?

Reply:
yeah umask has to be set in the shell that is running the subsequent commands... alternately we could create a shell script that has the umask in it first and the following commands.. but i'm not sure that buys us much

Reply:
Given that, I'm not sure why we bother splitting it up into separate arguments.

Reply:
if we don't then it tries to escape the space and umask fails.. <shrug>